integrations/src/test/resources/io/jenkins/plugins/casc/ViewJobFilters.yml

```diff
@@ -0,0 +1,31 @@
+jenkins:
```

Comment:
why is this file here? I don't think it's used

Reply:
oh, I referred to other tests and they had their yaml file here along with one at their readme.md. I'll delete it now.